packages/website/docs/context.md

```diff
@@ -50,3 +42,23 @@ const autocomplete = createAutocomplete({
   },
 });
 ```
+
+Context can be handy when developing Autocomplete plugins. It avoids polluting the global namespace while still being able to pass data around across different lifecycle hooks.
+
+## Reference
+
+The `setContext` function is accessible on your `autocomplete` instance. It's also provided in the `subscribe` function of Autocomplete plugins.
+
+The `context` object is available on the [`state`](/docs/state) object.
+
+### `setContext`
+
+> `<TState>(value: TState) => void`
```

Comment:
This is more correct and easier to understand. (It should have been `TState['context']` otherwise.)

```suggestion
> `(value: Record<string, unknown>) => void`
```

Reply:
Right, had it on my branch, forgot to push.